api/src/main/kotlin/com/gmtkgamejam/routing/PostRoutes.kt

```diff
@@ -23,87 +22,78 @@ fun Application.configurePostRouting() {
     val service = PostService()
     val favouritesService = FavouritesService()
 
+    fun getFilterFromParameters(params: Parameters): List<Bson> {
+        val filters = mutableListOf<Bson>(/*PostItem::deletedAt eq null*/)
```

Comment:
If `PostItem::deletedAt eq null` is commented out, this query will bring back deleted posts

Reply:
Oh forgot to revert after testing it out 🙈